INFO     self_developed_model.conftest:conftest.py:62 [2025-09-04 15:00:49] 用例 test_cloudTrader/test_delete.py::TestDelete_cloudTrader::test_delete_userlist 开始登录，环境: uat
INFO     self_developed_model.conftest:conftest.py:100 UAT登录尝试 1/5，MFA验证码: 385403
INFO     self_developed_model.conftest:conftest.py:124 UAT登录成功（第1次尝试）
INFO     self_developed_model.commons.variable_manager:variable_manager.py:111 [2025-09-04 15:00] 运行时变量已保存到: D:\pycharm_test\self_developed_model\VAR\runtime_vars_cloud.json
INFO     self_developed_model.commons.api_base:api_base.py:239 [2025-09-04 15:00:51] DELETE请求: /mascontrol/user | 数据: [7206]
INFO     root:test_delete.py:454 第1个账号（ID: 7206）删除接口调用成功
INFO     self_developed_model.commons.api_base:api_base.py:239 [2025-09-04 15:00:53] DELETE请求: /mascontrol/user | 数据: [7201]
INFO     root:test_delete.py:454 第2个账号（ID: 7201）删除接口调用成功
INFO     self_developed_model.commons.api_base:api_base.py:239 [2025-09-04 15:00:54] DELETE请求: /mascontrol/user | 数据: [7203]
INFO     root:test_delete.py:454 第3个账号（ID: 7203）删除接口调用成功
INFO     self_developed_model.commons.api_base:api_base.py:239 [2025-09-04 15:00:56] DELETE请求: /mascontrol/user | 数据: [7208]
INFO     root:test_delete.py:454 第4个账号（ID: 7208）删除接口调用成功
INFO     self_developed_model.commons.api_base:api_base.py:239 [2025-09-04 15:00:58] DELETE请求: /mascontrol/user | 数据: [7199]
INFO     root:test_delete.py:454 第5个账号（ID: 7199）删除接口调用成功
INFO     self_developed_model.commons.api_base:api_base.py:239 [2025-09-04 15:00:59] DELETE请求: /mascontrol/user | 数据: [7207]
INFO     root:test_delete.py:454 第6个账号（ID: 7207）删除接口调用成功
INFO     self_developed_model.commons.api_base:api_base.py:239 [2025-09-04 15:01:01] DELETE请求: /mascontrol/user | 数据: [7202]
INFO     root:test_delete.py:454 第7个账号（ID: 7202）删除接口调用成功
INFO     self_developed_model.commons.api_base:api_base.py:239 [2025-09-04 15:01:02] DELETE请求: /mascontrol/user | 数据: [7205]
INFO     root:test_delete.py:454 第8个账号（ID: 7205）删除接口调用成功
INFO     self_developed_model.commons.api_base:api_base.py:239 [2025-09-04 15:01:04] DELETE请求: /mascontrol/user | 数据: [7200]
INFO     root:test_delete.py:454 第9个账号（ID: 7200）删除接口调用成功
INFO     self_developed_model.commons.api_base:api_base.py:239 [2025-09-04 15:01:05] DELETE请求: /mascontrol/user | 数据: [7204]
INFO     root:test_delete.py:454 第10个账号（ID: 7204）删除接口调用成功
INFO     self_developed_model.conftest:conftest.py:157 [2025-09-04 15:01:06] 用例 test_cloudTrader/test_delete.py::TestDelete_cloudTrader::test_delete_userlist 执行完毕